### ISSUE TITLE
Accessibility - Umbraco Cloud - Profile Settings - There is no focus on buttons when tabbing through content.

### ISSUE THREAD

**Issue by Alex-Lovell** [umbraco/Umbraco.Cloud.Issues#616]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.3.0 of Cloud

### Bug summary

When I attempt to tab through the content on the Profile Settings page within Umbraco Cloud, there are several buttons that do not come into focus when the are selected.

### Specifics

https://www.s1.umbraco.io/profile

### Steps to reproduce

**STEPS:**

1. Login to the CMS with a valid account
2. Select 'Profile' from the dropdown in top right of the screen where you see your username
3. Tab through the content using keyboard only

![image](https://user-images.githubusercontent.com/98405431/199204363-4eb4699d-c961-4936-9ac9-1fafd097b78d.png)

### Expected result / actual result

**EXPECTED RESULT:** 
All buttons, links that acre active should come into focus in the same way that they would if you were hovering over them with a mouse.

**ACTUAL RESULT:** 
The following buttons (CTA's) do not have any focus at all:

- Site logo (although there is a line after it)
- Create New Project
- Edit Profile
- Change Password
- Chat icon

**Comment by RyuLindow** [comment]
Hey @Truealex4 

Thanks for the report. FYI I've moved it to the Umbraco Cloud issue tracker :)

**Comment by sajumb** [comment]
Hi @Truealex4 
Thanks for your fine report. You are correct, we should handle focus and accessibilities concern better on the "Profile" page in Umbraco Cloud Portal. And we will. All portal pages is been migrated to a new tech stack based on web components. Throughout the migration phase, we will ensure that we are aligned with accessibility principles and best practices. We will use your good report above as a checklist during the migration of the "Profile" page in particular and other pages in general.
